src/substrate/filters/type_parser.ts

```diff
@@ -198,7 +198,7 @@ export function ParseType(
           return EventKind.TreasuryBountyClaimed;
         case 'BountyCanceled':
           return EventKind.TreasuryBountyCanceled;
-        case 'BountyExtended':
+        case 'extendBountyExpiry':
```

Comment:
Do we know why some methods are CamelCase event names, and others are actual method names?

Reply:
The methods triggered by events on the chain are CamelCase, but the ones triggered by extrinsics are the method names.